janis_bioinformatics/tools/common/gatkbasecalbam.py

```diff
@@ -0,0 +1,60 @@
+from janis_bioinformatics.data_types import BamBai, FastaWithDict, VcfTabix, Bed
+from janis_bioinformatics.tools.gatk4 import (
+    Gatk4BaseRecalibrator_4_1_3,
+    Gatk4ApplyBqsr_4_1_3,
+)
+from janis_bioinformatics.tools.bioinformaticstoolbase import BioinformaticsWorkflow
+
+
+class GATKBaseRecalBam_4_1_3(BioinformaticsWorkflow):
```

Comment:
Per internal conversation, can we give this one a name that better describes what it does.

Reply:
fix in commit 
4633b67